contributor/making-release.md

```diff
@@ -6,7 +6,43 @@ redirect_from:
   - /governance/releasing
 ---
 
-This is a guide for the creation of a release of Apache Accumulo. 
+Follow these steps to make a release of Apache Accumulo. 
+
+1. [Setup](#setup)
+2. [Update the CHANGES file](#update-the-changes-file)
+3. [Triage Issues](#triage-issues)
+4. [Create the candidate](#create-the-candidate)
+5. [Vote](#vote)
+6. [Promote the artifacts](#promote-the-artifacts)
```

Comment:
I would probably create a "post-release tasks" section starting at after the vote.